src/jsMain/js/FakeClientDevice.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
+import Draggable from 'react-draggable';
+import styles from './FakeClientDevice.scss';
+
+export default class FakeClientDevice extends Component {
+  constructor(props) {
+    super(props);
+    this.width = props.width / 2;
```

Comment:
This is fine and dandy. An alternative way of writing this is by using a getter: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

An example is writing this like:

```jsx
export default class FakeClientDevice extends Component {
  get width() {
    return this.props.width / 2;
  }

  get height() {
    return this.props.height / 2;
  }

  ...
}
```

Reply:
Cool... the `/2` is sort of a vague stab at being able to resize the device, like maybe with the magnifying glass/zoom icons… thinking I'll leave it as-is for now until they're hooked up though.